README.md

```diff
@@ -35,12 +35,16 @@ Things to note:
 A common pattern with Docker containers is to configure programs using environment variables rather than command-line options. `pydexec` offers some tools for working with environment variables that can simplify configuring programs in this way:
 ```python
 cmd = Command('my-executable')
-(cmd.env_to_arg('FOO')                 # arg with value $FOO if set
-    .env_to_arg('BAR', default='bar')  # arg with value $FOO if set, else 'bar'
-    .env_to_opt('--foo', 'FOO')        # opt --foo with value $FOO if set
-    .env_to_opt('--baz', 'BAZ', required=True))
+(cmd.arg_from_env('FOO')                 # arg with value $FOO if set
+    .arg_from_env('BAR', default='bar')
+        # arg with value $FOO if set, else 'bar'
+    .opt_from_env('--foo', 'FOO')        # opt --foo with value $FOO if set
+    .opt_from_env('--baz', 'BAZ', required=True))
         # opt --baz with value $BAZ if set, else raise an error
 
+# Don't remove $ABC from the command's environment (the default behaviour)
```

Comment:
It's unclear whether "the default behaviour" in this case is removing `$ABC` or not removing `$ABC`. Maybe expand to comment to include something like "by default, `$ABC` is removed from the environment because it may contain the keys to all your databases".